src/components/Generic/Button/Button.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import Button from './Button';
+
+const className = 'test';
+const testChildren = 'test1';
+
+const setUp = () => mount(<Button className={className}>{testChildren}</Button>);
+
+describe('Button Tests.', () => {
+  let component;
+  beforeEach(() => {
+    component = setUp();
+  });
+
+  it('Btn className to be $className + "btn"', () => {
+    const classNameProp = component.find('.btn').instance().className;
+    expect(classNameProp).toBe(`${className} btn`);
+  });
+
+  it('Btn text to be children', () => {
+    const childrenProp = component.prop('children');
+    const childrenText = component.children().text();
```

Comment:
We don't need to get children text since we know it. We need to get button text.